### PR TITLE
skip flaky lightbox test

### PR DESCRIPTION
**Pull request by rileyajones**
This test has been a common source of flakiness in CI
http://amp-test-cases.appspot.com/test-results/history/71865af50987cd6ec686cf54939edcc0

#32442

### CODE REVIEW

File: extensions/amp-lightbox/1.0/test-e2e/test-custom-close-button.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ describes.endtoend(
       ).to.equal(0);
     });
 
-    it('should open the lightbox', async () => {
+    // TODO(wg-components, #24195): Flaky during CI.
```
Comment (rsimha) [inline extensions/amp-lightbox/1.0/test-e2e/test-custom-close-button.js:48]: #24195 tracks the carousel failures. There's a different issue for lightbox that can be used here: #28948.

**Review by rsimha — APPROVED** [review]
LGTM after fixing the TODO.

**Review by alanorozco — APPROVED** [review]

**Comment by raja-anbazhagan** [comment]
Guys, Would you mind taking a look at this [Experiment B Tests failure](https://circleci.com/gh/ampproject/amphtml/67668?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) in my PR#33383?

Looks like this one is one of those flaky e2e tests from amp-carousel component as my code has nothing to do with amp-carousel.

File: extensions/amp-lightbox/1.0/test-e2e/test-custom-close-button.js [hunk 2]
```diff
@@ -45,7 +45,8 @@ describes.endtoend(
       ).to.equal(0);
     });
 
-    it('should open the lightbox', async () => {
+    // TODO(wg-components, #28948): Flaky during CI.
+    it.skip('should open the lightbox', async () => {
```
Comment (rsimha) [inline extensions/amp-lightbox/1.0/test-e2e/test-custom-close-button.js:49]: This test is still red on `master`. Looks like the failure source is in `extensions/amp-lightbox/test-e2e/test-amp-lightbox.js`.

**Pull request merged**